.github/workflows/upload-poeditor.yml

```
@@ -0,0 +1,101 @@
+on:
+  pull_request:
+    types: [closed]
+    branches:
+      - develop
+    paths:
+      - "src/locales/**"
+
+jobs:
+  upload-translations:
+    # Only run if the PR was merged (not just closed) or manually triggered
+    if: github.event.pull_request.merged == true'
```

actionlint (1.7.4)

got unexpected EOF while lexing end of string literal, expecting '''

(expression)